scripts/vscode.patch

```diff
@@ -885,7 +894,7 @@ index 48ef482..dc47f81 100644
 -			placeHolder: isMacintosh ? nls.localize('openRecentPlaceHolderMac', "Select to open (hold Cmd-key to open in new window)") : nls.localize('openRecentPlaceHolder', "Select to open (hold Ctrl-key to open in new window)"),
 +			placeHolder: browser.isMacintosh ? nls.localize('openRecentPlaceHolderMac', "Select to open (hold Cmd-key to open in new window)") : nls.localize('openRecentPlaceHolder', "Select to open (hold Ctrl-key to open in new window)"),
 diff --git a/src/vs/workbench/electron-browser/main.contribution.ts b/src/vs/workbench/electron-browser/main.contribution.ts
-index 71bc992..111ce51 100644
+index 71bc992..2692f39 100644
```

Comment:
Looks like there's a conflict here with master that needs to be resolved. You could just revert this line, or merge and regenerate the patch. I'm not sure why it generated a different hash. :confused:

Reply:
fixed